spec/presenters/history_list_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe HistoryList do
+  describe 'contents' do
```

Comment:
usually in the `describe` blocks we include the name of the method we are testing. looks like this one is testing `events` so I'd have this as:

```ruby
describe "#events" do
  it "returns proposal creation first" do
    ...
  end
end
```